freqtrade/strategy/interface.py

```diff
@@ -134,6 +136,39 @@ def __init__(self, config: dict) -> None:
         self._last_candle_seen_per_pair: Dict[str, datetime] = {}
         super().__init__(config)
 
+        # Gather informative pairs from @informative-decorated methods.
+        self._ft_informative: Dict[
+            Tuple[str, str], Tuple[InformativeData,
+                                   Callable[[Any, DataFrame, dict], DataFrame]]] = {}
```

Comment:
Please extract `Callable[[Any, DataFrame, dict], DataFrame]]` and assign it a named type. 
it's used 3 times in total (which justifies extraction) - and this type-def alone scares me from looking at the rest :laughing: